src/animad-keepers-view.html

```diff
@@ -38,14 +49,43 @@ <h2>Beispiel Filtern ohne Tabellenbearbeitungs-Header</h2>
         <animad-keeper-create-form></animad-keeper-create-form>
       </div>
 
-      </div>
+    </div>
+  -->
   </template>
 
   <script>
     // GENERATOR pattern (camel case): [DOMAINNAME][ENTITYNAME]View
     class AnimadKeepersView extends Polymer.Element {
       // GENERATOR pattern: [DOMAINNAME]-[ENTITYNAME]-view
       static get is() { return 'animad-keepers-view'; }
+
+      static get properties() {
```

Comment:
Brauchen wir das in jeder view? Falls ja, würde es sich dann nicht lohnen ein Mixin zu machen?

Reply:
Auf was bezieht sich der Kommentar? Auf die iron-pages in animad-keepers-view? Die Werte für den Aufruf der Seiten zumindest bei der Liste können aber schon sehr unterschiedlich sein (Sofort alle Daten laden ja/nein, Header in der Liste anzeigen ja/nein, etc).

Reply:
Der bezieht sich auf diese ganzen Properties - also blöderweise alles was man gerade nicht sieht, weil ich den Kommentar ganz oben hin gemacht habe ;)

Reply:
Ach so. Ja, macht wahrscheinlich Sinn, weil die gleichen Properties gibt's auch in animad-app und den anderen views.